mk/package-and-sign.sh

```diff
@@ -64,8 +63,8 @@ echo "INFO: Build and sign the installers..."
 . ${REPO}/mk/build-installers.sh
 
 #collect output and extra files to the OUTPUT_DIR
-cp ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.exe ${OUTPUT_DIR}
-cp ${DOTNETINST}/${BRANDING_BRAND_CONSOLE}Setup.l10n.exe ${OUTPUT_DIR}
+cp ${WIX}/out${BRANDING_BRAND_CONSOLE}/${BRANDING_BRAND_CONSOLE}.msi ${OUTPUT_DIR}
```

Comment:
These files are already copied the the OUTPUT_DIR at the end of build-installers.sh script. We should only do it once.